phpunit.xml.dist

```diff
@@ -54,6 +54,7 @@
 
     <listeners>
         <listener class="Doctrine\Tests\DbalPerformanceTestListener"/>
+        <listener class="Symfony\Bridge\PhpUnit\SymfonyTestsListener" />
```

Comment:
This is how you get to test deprecations with annotations

Reply:
Not a fan of this, but hey, it's dev-only, so I can live with it.